src/Himali_Assignment28/Assignment28.java

```diff
@@ -0,0 +1,45 @@
+/*find the triplet having maximum sum.
+[10,12,8,20,18,22,15,13,17]
+output : maximum sum triplet 20,18,22 --> 60*/
+
+package Himali_Assignment28;
+
+import java.util.Arrays;
+
+public class Assignment28 {
+	
+	void tripletSum(int array[]) {
+		int maxSum=0;
+		int[] triplet=new int[3];
+		
+			for(int i=0;i<(array.length)-2;i++) {
```

Comment:
this code not give expected output, check for this input:
3, 4, 5, 6, 352, 1, 4, 69, 90
it should print 6, 352, 1but your code printing 5,6,352
consecutives are getting printed